nilearn/plotting/displays/_slicers.py

```diff
@@ -1816,7 +1818,7 @@ class MosaicSlicer(BaseSlicer):
 
     _cut_displayed = "yxz"
     _axes_class = CutAxes
-    _default_figsize = [11.1, 7.2]
+    _default_figsize = [11.1, 20.0]
```

Comment:
this seems like a large increase, why is it necessary?

Reply:
> this seems like a large increase, why is it necessary?

It was hard to get the title not to overlap and then to me the rendering in the examples seemed better since the brains are bigger and closer together so less white space. See the mosaic plots in this example ([old](https://nilearn.github.io/stable/auto_examples/01_plotting/plot_demo_more_plotting.html#sphx-glr-auto-examples-01-plotting-plot-demo-more-plotting-py) vs [new](https://output.circle-artifacts.com/output/job/cde255e7-0a62-4aff-8607-6a8301a138ec/artifacts/0/dev/auto_examples/01_plotting/plot_demo_more_plotting.html#sphx-glr-auto-examples-01-plotting-plot-demo-more-plotting-py)).